.circleci/config.yml

```diff
@@ -3,140 +3,363 @@
 
 version: 2.1
 
+orbs:
+  codecov: codecov/codecov@3.2.4
+  macos: circleci/macos@2
+  
 # Workflows orchestrate a set of jobs to be run;
 workflows:
   version: 2
   build-test:
     jobs:
-      - build-and-test
+      - validate-code
+      - test-ios-core:
+          requires:
+            - validate-code
```

Comment:
Can we remove this from each job and just run it once? I see that `validate-code` runs lint for all the extensions.

Reply:
@praveek Yeah that only runs once. The `requires` is there to let the workflow know that it needs to run before all the other jobs run.